Add ?platform for /experiments query (#283)
discord-userdoccers/discord-userdoccers: pages/topics/experiments.mdx

###### Query String Parameters

| Field                   | Type    | Description                                                           |
| ----------------------- | ------- | --------------------------------------------------------------------- |
| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data             |

> | platform?               | string  | Includes experiments from provided platform enum in the returned data |

Reviewer: ```suggestion
| platform?               | string  | Limits the response to experiments targeting the selected platform |
```

> | platform?               | string  | Includes experiments from provided platform enum in the returned data |

Reviewer 2: ```suggestion
| platform?               | string  | Supplements the response with experiments from the given [platform](#experiment-platform) |

###### Experiment Platform

| Value | Description |
| ------ | ------------ |
| DEVELOPER_PORTAL | The developer portal |
```

###### Query String Parameters

| Field                   | Type    | Description                                                           |
| ----------------------- | ------- | --------------------------------------------------------------------- |
| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data             |

> | platform?               | string  | Supplements the response with experiments from the given [platform](#experiment-platform) |

Reviewer: ```suggestion
| platform?               | string  | Whether to also include experiments for the given [platform](#experiment-platform) |
```

It's still unclear to me what the param actually does... 

Maybe worth it to add a sentence outside the table to clarify in what situation it's sent and why the data isn't included by default?
